docker/web/Dockerfile

```
@@ -0,0 +1,3 @@
+    FROM nginx
+
+    COPY ./build/prod /usr/share/nginx/html
```

```suggestion
FROM nginx

COPY ./build/prod /usr/share/nginx/html
```

The indentation is not necessary.